django_app/settings.py

```diff
@@ -11,13 +11,15 @@
 """
 
 import os
+import os.path
```

Comment:
No need to extra import because you already import OS, so you can directly use os.path.